Boccia-Unity/Assets/Boccia/UI/GameOptionsMenuPresenter.cs

```diff
@@ -15,125 +15,134 @@ public class GameOptionsMenuPresenter : MonoBehaviour
     public Slider rotationSpeedSlider;
     private BocciaModel model;
     public Button doneButton;
-
-    private static readonly Dictionary<string, Color> colors = new Dictionary<string, Color>
-    {
-        {"Blue", Color.blue },
-        {"Red", Color.red },
-        {"Green", Color.green },
-        {"Yellow", Color.yellow },
-        {"Black", Color.black },
-        {"Magenta", Color.magenta},
-        {"Grey", Color.grey},
-        {"Cyan", Color.cyan}
-    };
```

Comment:
You defined a dictionary here and in Boccia model, wouldn't it be better to have a single dictionary on a namespace and have both classes access that?